picard/coverart/providers/caa.py

```diff
@@ -46,6 +46,61 @@
 
 _CAA_IMAGE_SIZE_DEFAULT = 500
 
+_CAA_IMAGE_TYPE_DEFAULT_INCLUDE = [ 'front', ]
+_CAA_IMAGE_TYPE_DEFAULT_EXCLUDE = [ 'raw/unedited', 'watermark', ]
+
+
+def make_list_box():
+    """Make standard list box for CAA image type selection dialog."""
+    list_box = QtWidgets.QListWidget()
+    list_box.setFixedSize(QtCore.QSize(150, 250))
+    list_box.setSizePolicy(QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
+    list_box.setSortingEnabled(True)
+    list_box.setSelectionMode(QtWidgets.QAbstractItemView.ExtendedSelection)
+    return list_box
```

Comment:
Instead of using `make_*()`, you could subclass Qt objects

```python
class ListBox(QtWidgets.QListWidget):
    def __init__(self, parent=None):
        super().__init__(parent=parent)
        self.setFixedSize(QtCore.QSize(150, 250))
        self.setSizePolicy(QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
        self.setSortingEnabled(True)
        self.setSelectionMode(QtWidgets.QAbstractItemView.ExtendedSelection)
```

```python
class ArrowButton(QtWidgets.QPushButton):
    def __init__(self, text, parent=None, command=None):
        super().__init__(text, parent=parent)
        if command:
            self.clicked.connect(command)
        self.setFixedSize(QtCore.QSize(35, 20))
```

And same for `make_arrows_column()`

Reply:
Makes sense.  It seems I haven't completely embraced the concept of OOP yet.  Old habits die hard.  :grin: